script.js

```diff
@@ -0,0 +1,480 @@
+
+const Keyboard = {
+  elements: {
+
+    mainContainer: null,
+    keysContainer: null,
+    keys: [],
+    keysEng: ['`', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace', 'Tab', 'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', ']', '\\', 'CapsLock', 'A', 'S', 'D', 'F', 'G', 'H', 'J', 'K', 'L', ';', "'", 'Enter', 'ShiftLeft', 'Z', 'X', 'C', 'V', 'B', 'N', 'M', ',', '.', '/', 'ArrowUp', 'ShiftRight', 'ControlL', 'Meta', 'AltL', ' ', 'AltR', 'ControlR', 'ArrowLeft', 'ArrowDown', 'ArrowRight'],
+    ruKeys: ['`', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace', 'Tab', 'й', 'ц', 'у', 'к', 'е', 'н', 'г', 'ш', 'щ', 'з', 'х', 'ъ', '\\', 'CapsLock', 'ф', 'ы', 'в', 'а', 'п', 'р', 'о', 'л', 'д', 'ж', 'э', 'Enter', 'ShiftLeft', 'я', 'ч', 'с', 'м', 'и', 'т', 'ь', 'б', 'ю', '/', 'ArrowUp', 'ShiftRight', 'ControlL', 'Meta', 'AltL', ' ', 'AltR', 'ControlR', 'ArrowLeft', 'ArrowDown', 'ArrowRight'],
+  },
+
+  eventHandler: {
+    oninput: null,
+  },
+
+  properties: {
+    value: '',
+    capsLock: false,
+    eng: true,
+    shift1: false,
+    alt1: false,
+  },
+
+  checkSessionStorage() {
+    if (sessionStorage.getItem('capsLock') === 'true') {
+      this.properties.capsLock = true;
+    }
+    if (sessionStorage.getItem('eng') === 'false') {
+      this.properties.eng = false;
+    }
+  },
+  init() {
+    // creat elements
+    const description = document.createElement('div');
+    const textarea = document.createElement('textarea');
+    this.elements.mainContainer = document.createElement('div');
+    this.elements.keysContainer = document.createElement('div');
+
+    // add classes to the elements
+    textarea.classList.add('textarea');
+    description.classList.add('description');
+
+
+    /* document.querySelector('.textarea').innerHTML */
+    this.elements.mainContainer.classList.add('keyboard');
+    this.elements.keysContainer.classList.add('keyboard_keys');
+    this.elements.keysContainer.appendChild(this.createKeys());
+    document.body.appendChild(description);
+    description.innerHTML = 'Клавиатура создана на WINDOWS</br>Для измения языка, нажмите сочетание клавиш Shift + Alt.';
+
+    this.elements.keys = this.elements.keysContainer.querySelectorAll('.keyboard_key');
+
+    // add to DOM
+    document.body.appendChild(textarea);
+    document.body.appendChild(this.elements.mainContainer);
```

Comment:
Yoy add here some elements to DOM and on the 48 line too. It hard to read such code. Try to keep it clean and structured. Do you aggre that the code below is better to read?
```suggestion
    //create description element
     const description = document.createElement('div');
      description.classList.add('description');
     description.innerHTML = 'Клавиатура создана на WINDOWS</br>Для измения языка, нажмите сочетание клавиш Shift + Alt.';
    
    // creat textarea and make all it needs
    const textarea = document.createElement('textarea');
    textarea.classList.add('textarea');
    textarea.placeholder = 'Введите текст...';
    
    //create main container
     this.elements.mainContainer.classList.add('keyboard');
     this.elements.mainContainer = document.createElement('div');
     
     //create keys container
    this.elements.keysContainer = document.createElement('div');
    this.elements.keysContainer.classList.add('keyboard_keys');
    this.elements.keysContainer.appendChild(this.createKeys());
     this.elements.mainContainer.appendChild(this.elements.keysContainer);
   
    // add to DOM everything in the end
    document.body.appendChild(description);
    document.body.appendChild(textarea);
    document.body.appendChild(this.elements.mainContainer);

```